doc/theoretical_description_multilabel_classification.rst

```diff
@@ -7,16 +7,16 @@ Theoretical Description
 =======================
 
 
-Two methods for multi-label uncertainty-quantification have been implemented in MAPIE so far :
-Risk-Controlling Prediction Sets (RCPS) [1] and Conformal Risk Control (CRC) [2].
+Three methods for multi-label uncertainty-quantification have been implemented in MAPIE so far :
+Risk-Controlling Prediction Sets (RCPS) [1], Conformal Risk Control (CRC) [2] and Learn Then Test (LTT) [3].
 The difference between these methods is the way the conformity scores are computed. 
 
 For a multi-label classification problem in a standard independent and identically distributed (i.i.d) case,
 our training data :math:`(X, Y) = \{(x_1, y_1), \ldots, (x_n, y_n)\}`` has an unknown distribution :math:`P_{X, Y}`. 
 
 For any risk level :math:`\alpha` between 0 and 1, the methods implemented in MAPIE allow the user to construct a prediction
 set :math:`\hat{C}_{n, \alpha}(X_{n+1})` for a new observation :math:`\left( X_{n+1},Y_{n+1} \right)` with a guarantee
-on the recall. RCPS and CRC give two slightly different guarantees:
+on the recall or the precision. RCPS, LTT and CRC give three slightly different guarantees:
```

Comment:
UPD: it's not clear whether RCPS and CRC only concern recall and LTT both precision and recall.

Reply:
Is the update satisfying ?

Reply:
Thank you for this update :)